mapstore/new.json

```diff
@@ -0,0 +1,35 @@
+{
+  "map": {
+    "projection": "EPSG:3857",
+    "units": "m",
+    "center": {
+      "x": 315445.640682,
+      "y": 5971955.5,
+      "crs": "EPSG:3857"
+    },
+    "zoom": 6,
+    "maxExtent": [
+      -20037508.34,
+      -20037508.34,
+      20037508.34,
+      20037508.34
+    ],
+    "layers": [
+      {
+        "type": "osm",
+        "title": "Open Street Map",
```

Comment:
OpenStreetMap is the official name.

Reply:
[this](https://github.com/georchestra/datadir/pull/186/commits/486b73c1a61673a52ca2ccc1603478405a1c49ec) is correction for offical OpenStreetMap's name.